### DISCUSSION TITLE
Welcome to Scenic Discussions!

### DISCUSSION THREAD

**Discussion by derekprior** [scenic-views/scenic#347] [Announcements]
## 👋 Welcome!

We're using Discussions as a place to:

* 💡 Discuss feature ideas prior to implementation
* 🐛 Troubleshoot issues with Scenic prior to opening any issues
* ❓ Ask questions about best practices for Scenic and around view usage in Rails apps in general
* 📣 Announce any major releases or changes

While I don't expect this to be a very active discussion forum, I do hope that using discussions can help keep the project's issues to actionable items while also helping y'all get the answers you need.

**Comment by edsondario** [comment]
I did according to the gem manual, but how do I show the view created? I apologize for the stupidity.
